apps/code-examples/src/app/code-examples/lookup/lookup/multi-select/lookup-multiple-demo.component.ts

```diff
@@ -51,7 +54,14 @@ export class LookupMultipleSelectDemoComponent implements OnInit {
   public getSearchFilters(): SkyAutocompleteSearchFunctionFilter[] {
     const names: any[] = this.myForm.controls.names.value;
     return [
-      (searchText: string, item: any): boolean => {
+      (
+        searchText: string,
+        item: any,
+        args?: SkyAutocompleteSearchArgs
+      ): boolean => {
+        if (args?.context === 'modal') {
```

Comment:
Is this a real use-case we want to communicate to customers? If so, do you mind putting a comment block above explaining what it's doing?

Reply:
Yes, it is. Done in [add comment explaining search function filter](https://github.com/blackbaud/skyux/pull/381/commits/83d9a2e237c84c9977814688e58770e2e68824ee)